python-library/fullservice.py

```diff
@@ -110,15 +110,15 @@ async def build_and_submit_transaction(
 
     async def build_burn_transaction(
         self,
-        account_id,
-        amount={"value": "", "token_id": ""},
-        redemption_memo_hex="",
-        input_txo_ids="",
-        fee_value="",
-        fee_token_id="",
-        tombstone_block="",
-        max_spendable_value="",
-    ):
+        account_id: str,
```

Comment:
I think you might as well add the `str` type to all the rest of the `account_id` parameters in this file.

Reply:
I'm actually using [monkeytype](https://github.com/Instagram/MonkeyType) to dynamically annotate the types of functions as they're called. I'll see about annotating all of the account id params though!